webpack.config.ts

```diff
@@ -3,13 +3,22 @@ import WarningsToErrorsPlugin from "warnings-to-errors-webpack-plugin";
 
 import path from "path";
 
+type EntryType = {
```

Comment:
I found c8 code coverage did not work well when js files are bundled.  So for running UI-tests with code coverage, suppress creating bundles of main server/client js files. Still syntaxHighlighter and webview js files need to be built as bundled js files.